### DISCUSSION TITLE
How to run external shell command on selected text?

### DISCUSSION THREAD

**Discussion by kljensen** [zed-industries/zed#16599] [Feature Requests]
I often do this in other editors. I couldn't find how to do it in Zed.

I want to select text and, e.g. run `fmt` on it or some `sed` or `awk` command.

**Comment by failable** [comment]
https://zed.dev/docs/tasks

But you can't insert back the command output to the buffer.

**Reply by kljensen** [reply]
Thanks. Hoping to see this feature

**Reply by failable** [reply]
Please upvote for the issue :) https://github.com/zed-industries/zed/issues/12598
